sdk/storage/Azure.Storage.Blobs.AspNetCore.DataProtection/src/Azure.Storage.Blobs.AspNetCore.DataProtection.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <Description>Microsoft Azure Blob storage support as key store.</Description>
+    <PackageTags>aspnetcore;dataprotection;azure;blob</PackageTags>
+    <Version>1.0.0-preview.1</Version>
+    <EnableApiCompat>false</EnableApiCompat>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.DataProtection" />
```

Comment:
Have we received an official stamp of approval for this dependency?

Reply:
Haven't asked for it, seems like if we are shipping a plugin for something we got to be able to reference that something.

Reply:
We've had this discussion with other partners (i.e, I'm shipping an Avro feature so I've got to be able to reference the Avro parser) and it'd be good if our team [followed the same rules](https://azure.github.io/azure-sdk/general_implementation.html#general-dependencies-approved-only).  I can't imagine you wouldn't get it approved though.

Reply:
Sounds good.

Reply:
Would you prefer getting the approval before merging the PR?

Reply:
No, so long as @KrzysztofCwalina gives a thumbs up before we ship any Previews.